### PR TITLE
Add pricing v2 reports apis

### PR DESCRIPTION
**Pull request by evanluc**
Adding API docs for the v2 reports.

### CODE REVIEW

File: source/includes/administration/_reports_pricing_v2.md [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+}
+
+```
+
+Organization Report Attributes | &nbsp;
+---------- | -----------
+`currencies`<br/>*Array[Object]* | Returns an array of currencies for the usage.
+`currencies.currency`<br/>*string* | The short-name of the currency.
+`currencies.total`<br/>*string* | A string containing the total of all usage.
+`currencies.categories`<br/>*Array[Object]* | A array of category objects. Contains all categories that had usage for the period.
```
Comment (patrickspensieri) [inline source/includes/administration/_reports_pricing_v2.md]: An array

File: source/includes/administration/_reports_pricing_v2.md [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+`endDate`<br/>*string* | An ISO-8601 instant format string representing the end of the report.
+`reportGenerated`<br/>*boolean* | Whether or not a report could be generated for this time period.
+
+<!------------------- GET CUSTOMER REPORT --------------------->
+
+### Get a customer's report
+
+`GET /reports/customers?start_date:=end_date=&end_date=:end_date`
+
+Retrieves a list of usage all usages for the provided reseller organization and all of it's suborgs.
```
Comment (patrickspensieri) [inline source/includes/administration/_reports_pricing_v2.md]: Retrieves a list of all usages

File: source/includes/administration/_reports_pricing_v2.md [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+  }
+}
+```
+
+Customer Report Attributes | &nbsp;
+---------- | -----------
+`organizations`<br/>*Array[Object]* | Returns an array of organizations for the usage. There will be one entry per organization and applied pricing combination found for the time period.
+`organizations.currency`<br/>*string* | The short-name of the currency.
+`organizations.total`<br/>*string* | A string containing the total of all usage.
+`organizations.categories`<br/>*Array[Object]* | A array of category objects. Contains all categories that had usage for the period.
```
Comment (patrickspensieri) [inline source/includes/administration/_reports_pricing_v2.md]: Same here, `An array`

File: source/includes/administration/_reports_pricing_v2.md [hunk 4]
```diff
@@ -0,0 +1,178 @@
+## Reports
+
+Reports using the pricing v2 engine
```
Comment (patrickspensieri) [inline source/includes/administration/_reports_pricing_v2.md]: Missing a period

File: source/includes/administration/_reports_pricing_v2.md [hunk 5]
```diff
@@ -0,0 +1,178 @@
...
+
+<!------------------- GET ORGANIZATION REPORT --------------------->
+
+### Get an organization report
+
+`GET /reports/organization_pricing?organization_id=:id&start_date:=end_date=&end_date=:end_date`
+
+Retrieves a list of usage grouped by currency and containing categories and their products. There is a cost sum at the currency, category and product level of granularity. This will create a report using applied pricings from the closest reseller to the organization.
```
Comment (patrickspensieri) [inline source/includes/administration/_reports_pricing_v2.md]: Maybe **Generates a report** rather than **This will create a report**?

**Review by patrickspensieri — COMMENTED** [review]
Minor grammar things

**Review by GuillaumeVialle — APPROVED** [review]
Looks good besides pats comments

File: source/includes/administration/_reports_pricing_v2.md (hunk 5 above)
Comment (evanluc) [reply source/includes/administration/_reports_pricing_v2.md]: 6c2e3a9f91f7e73660ad634a3263df65853ce5ee

File: source/includes/administration/_reports_pricing_v2.md (hunk 4 above)
Comment (evanluc) [reply source/includes/administration/_reports_pricing_v2.md]: 6c2e3a9f91f7e73660ad634a3263df65853ce5ee

File: source/includes/administration/_reports_pricing_v2.md (hunk 3 above)
Comment (evanluc) [reply source/includes/administration/_reports_pricing_v2.md]: 6c2e3a9f91f7e73660ad634a3263df65853ce5ee

File: source/includes/administration/_reports_pricing_v2.md (hunk 2 above)
Comment (evanluc) [reply source/includes/administration/_reports_pricing_v2.md]: 6c2e3a9f91f7e73660ad634a3263df65853ce5ee

File: source/includes/administration/_reports_pricing_v2.md (hunk 1 above)
Comment (evanluc) [reply source/includes/administration/_reports_pricing_v2.md]: 6c2e3a9f91f7e73660ad634a3263df65853ce5ee

**Pull request merged**